### PR TITLE
Backtracking

### PR DESCRIPTION
**Pull request by Frenkiee**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -93,6 +94,31 @@ services:
     networks:
       - reefscan
 
+  backtracking:
+    image: reef-explorer-backtracking-dev
```
Comment (frisitano) [inline docker-compose.yml]: We want to reuse the image from the crawler for this service.  Therefore, we should change `reef-explorer-backtracking-dev` to `reef-explorer-crawler-dev`

File: docker-compose.yml [hunk 2]
```diff
@@ -93,6 +94,31 @@ services:
...
+    command: yarn dev-backtracking
+    build:
```
Comment (frisitano) [inline docker-compose.yml]: we should be reusing image from crawler (`reef-explorer-crawler-dev`) so no need to build here, should be fine to remove `build`.

File: crawler/src/queries/evmEvent.ts [hunk 3]
```diff
@@ -20,126 +23,146 @@ const contractToValues = ({
...
+      raw: { address, topics, data }, parsed: {}, status: 'Success', type: 'Unverified',
+    };
     const contract = await getContractDB(address);
     if (contract.length === 0) {
       return evmData;
     }
     const iface = new ethersUtils.Interface(contract[0].compiled_data[contract[0].name]);
     try {
       evmData.parsed = iface.parseLog({ topics, data });
+      eventData.type = 'Verified';
```
Comment (matjazonline) [inline crawler/src/queries/evmEvent.ts]: evmData not eventData

File: crawler/src/queries/evmEvent.ts [hunk 4]
```diff
@@ -20,126 +23,146 @@ const contractToValues = ({
...
     const { topics, data } : BytecodeLog = eventData[0];
     let { address } : BytecodeLog = eventData[0];
     address = toContractAddress(address);
-    if(!address) {
+    if (!address) {
       return undefined;
     }
-    let evmData = {raw: {address, topics, data}, parsed: {}};
+    const evmData: CompleteEvmData = {
+      raw: { address, topics, data }, parsed: {}, status: 'Success', type: 'Unverified',
```
Comment (matjazonline) [inline crawler/src/queries/evmEvent.ts:72]: - we have `method` saved and status can be seen there 
- type can be seen from data_parsed===null

**Review by matjazonline — CHANGES_REQUESTED** [review]

File: crawler/src/queries/evmEvent.ts (hunk 4 above)
Comment (Frenkiee) [reply crawler/src/queries/evmEvent.ts:72]: for status ye we can remove it, but type should stay

**Review by frisitano — APPROVED** [review]

**Pull request merged**